config/pipeline.yaml

```diff
@@ -622,6 +622,96 @@ jobs:
       job_timeout: 10
     kcidb_test_suite: kselftest.iommu
 
+  preempt-rt: &preempt-rt
+    template: preempt-rt.jinja2
+    kind: test
```

Comment:
Is this a single test or a test suite?
In case of test suite, it should have `kind=job`.

Reply:
It is a test suite. I've updated the kind to job.